### PR TITLE
Feature/test delete salesdashboard

### PR DESCRIPTION
**Pull request by Lex-23**

### CODE REVIEW

File: cryptostock/account/tests/test_delete_salesdashboard.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from account.tests.factory import BrokerFactory, SalesDashboardFactory
+
+
+def test_delete_sales_dashboard(auth_broker, broker_account):
+    sale = SalesDashboardFactory(broker=broker_account)
+
+    response = auth_broker.delete(f"/api/salesdashboard/{sale.id}/")
+    response1 = auth_broker.get(f"/api/salesdashboard/{sale.id}/")
```
Comment (vadim-denisov) [inline cryptostock/account/tests/test_delete_salesdashboard.py:8]: Should check DB instead of API

Comment (Lex-23) [reply cryptostock/account/tests/test_delete_salesdashboard.py]: done

File: cryptostock/account/tests/test_delete_salesdashboard.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from django.http.response import Http404
+from rest_framework.generics import get_object_or_404
+
+
+def test_delete_sales_dashboard(auth_broker, broker_account):
+    sale = SalesDashboardFactory(broker=broker_account)
+
+    response = auth_broker.delete(f"/api/salesdashboard/{sale.id}/")
+    with pytest.raises(Http404) as exc:
+        get_object_or_404(SalesDashboard.objects.all(), id=sale.id)
```
Comment (vadim-denisov) [inline cryptostock/account/tests/test_delete_salesdashboard.py:13]: get_object_or_404 is shortcut. SalesDashboard.objects.get is better.

File: cryptostock/account/tests/test_delete_salesdashboard.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    assert response.status_code == 204
+    assert exc.type == Http404
```
Comment (vadim-denisov) [inline cryptostock/account/tests/test_delete_salesdashboard.py:16]: You already check type in line 12

File: cryptostock/account/tests/test_delete_salesdashboard.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    assert "No SalesDashboard matches the given query." in str(exc.value)
```
Comment (vadim-denisov) [inline cryptostock/account/tests/test_delete_salesdashboard.py:17]: Read the doc: https://docs.pytest.org/en/6.2.x/reference.html#pytest-raises

File: cryptostock/account/tests/test_delete_salesdashboard.py (hunk 2 above)
Comment (Lex-23) [reply cryptostock/account/tests/test_delete_salesdashboard.py]: done

File: cryptostock/account/tests/test_delete_salesdashboard.py (hunk 3 above)
Comment (Lex-23) [reply cryptostock/account/tests/test_delete_salesdashboard.py]: got it

File: cryptostock/account/tests/test_delete_salesdashboard.py (hunk 4 above)
Comment (Lex-23) [reply cryptostock/account/tests/test_delete_salesdashboard.py]: done

**Review by vadim-denisov — APPROVED** [review]

**Comment by Lex-23** [comment]
[write pytests](https://trello.com/c/KrUqCZfN/6-write-pytests)

**Pull request merged**